extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -136,28 +140,39 @@ export class AmpUserNotification extends AMP.BaseElement {
   }
 
   /**
+   * Constructs the new href to execute a `GET` request with the
+   * `elementId` and `ampUserId` query params appended.
+   * @param {string} ampUserId
+   * @return {!Promise<string>}
+   * @private
+   */
+  buildGetHref_(ampUserId) {
+    return this.urlReplacements_.expand(this.showIfHref_).then(href => {
+      const hrefObj = parseUrl(href);
+      const fields = `elementId=${encodeURIComponent(this.elementId_)}&` +
```

Comment:
One note. This is certainly fine. But, fyi, I just added a way to add custom parameters to urlReplacements. It could look like this:
```
urlReplaces.expand('?ts=TIMESTAMP&eid=NOTIFICATION_ID&...', {
  'NOTIFICATION_ID': 'my value here'
})
```